app/models/concerns/foreman_remote_execution/encrypted_value_extensions.rb

```diff
@@ -0,0 +1,22 @@
+module ForemanRemoteExecution
```

Comment:
Any reason we can't use the `Encryptable` module from the Foreman? https://github.com/theforeman/foreman/blob/b5eefd928921ddab7fce2b03882e1b8afa5f37f5/app/models/concerns/encryptable.rb#L2